bash_completion.d/000_bash_completion_compat.bash

```diff
@@ -235,4 +235,18 @@ _init_completion()
 # shellcheck disable=SC2154  # defined in the main "bash_completion"
 _backup_glob=$_comp_backup_glob
 
+# @deprecated Use "_comp_command_offset" instead.  It should be noted that the
+# new interface "_comp_command_offset" is changed to receive an index in
+# "words" instead of that in "COMP_WORDS" as "_command_offset" did.
```

Comment:
```suggestion
# @deprecated Use `_comp_command_offset` instead.  Note that the
# new interface `_comp_command_offset` is changed to receive an index in
# `words` instead of that in `COMP_WORDS` as `_command_offset` did.
```